torch/csrc/cuda/Module.cpp

```diff
@@ -630,6 +630,18 @@ PyObject * THCPModule_getCurrentBlasHandle_wrap(PyObject *self, PyObject *noargs
   END_HANDLE_TH_ERRORS
 }
 
+PyObject * THCPModule_rocm_is_backward_pass(PyObject *_unused, PyObject *noargs)
+{
+  HANDLE_TH_ERRORS
+#if USE_ROCM
+  bool value = at::BackwardPassGuard::is_backward_pass();
+#else
+  bool value = false;
+#endif
+  return PyBool_FromLong(value);
```

Comment:
nit: you can use `Py_RETURN_TRUE` and `Py_RETURN_FALSE` to avoid the costly conversion:
```cpp
if (at::BackwardPassGuard::is_backward_pass()) {
  Py_RETURN_TRUE;
else {
  Py_RETURN_FALSE;
}

```

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/71881/commits/7e18386a8983f825d26b6b2c425984c4af7a8ae7.